### ISSUE TITLE
Any plans to resume updates?

### ISSUE THREAD

**Issue by typedrat** [wyattowalsh/nbadb#9]
It would be incredibly helpful if this was updated on Kaggle again with more recent data.

**Comment by wyattowalsh** [comment]
yep, I'm in the process of overhauling the project and should have a new version with better data coverage soon (~1mo)
